src/wp-includes/rest-api/endpoints/class-wp-rest-global-styles-controller.php

```diff
@@ -10,24 +10,20 @@
 /**
  * Base Global Styles REST API Controller.
  */
-class WP_REST_Global_Styles_Controller extends WP_REST_Controller {
-
-	/**
-	 * Post type.
-	 *
-	 * @since 5.9.0
-	 * @var string
-	 */
-	protected $post_type;
-
+class WP_REST_Global_Styles_Controller extends WP_REST_Posts_Controller {
 	/**
```

Comment:
I think we should add 
```
protected $allow_batch = array( 'v1' => false );
```

To directly denote this endpoint does not support batching. This should be added to the register of the routes as well. See 
https://github.com/WordPress/wordpress-develop/blob/27087b71e0c109af68dbc0545893217fc80350db/src/wp-includes/rest-api/endpoints/class-wp-rest-posts-controller.php#L91

This means it could be overridden later in a plugin.

Reply:
Good spotting!